{"uuid": "7d13ae89-8917-4b8d-8942-6b00dad1a790", "children": ["940e400e-f34a-4ef8-9865-bc451426ab30"], "befores": [{"name": "pre_conditions", "status": "broken", "statusDetails": {"message": "TypeError: can only concatenate str (not \"int\") to str\n", "trace": "  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\aldo.madrid\\Documents\\AutomationExample\\Xirgo_KB1_Scripts\\Panic\\test_003_panic_and_power_loss.py\", line 11, in pre_conditions\n    Report.comment('***************Pre Conditions***************')\n  File \"C:\\Users\\aldo.madrid\\Documents\\AutomationExample\\Libraries\\report.py\", line 15, in comment\n    ident = '     ' + len(str(Report.__step_no)) + '  ' if Report.__step_active and not no_ident else ''\n"}, "start": 1637887296114, "stop": 1637887296114}], "start": 1637887296114, "stop": 1637887296123}